docs/conf.py

```diff
@@ -81,6 +82,13 @@
 version = get_version("zarr")
 release = get_version("zarr")
 
+redirects = {
+     "docs/spec": "https://zarr-specs.readthedocs.io",
```

Comment:
I'm struggling to test these under https://zarr--2555.org.readthedocs.build/en/2555/

https://zarr--2555.org.readthedocs.build/en/2555/spec/index.html e.g. is 404

Reply:
Should be working now - the redirect is from https://zarr--2555.org.readthedocs.build/en/2555/spec.html (corresponding to https://zarr.readthedocs.io/en/stable/spec.html).